operator/src/main/java/oracle/kubernetes/operator/rest/RestBackendImpl.java

```diff
@@ -417,6 +450,15 @@ private WebApplicationException createWebApplicationException(int status, String
     return new WebApplicationException(rb.build());
   }
 
+  protected boolean authenticateWithTokenReview() {
+    return "true".equalsIgnoreCase(Optional.ofNullable(getEnvVariable.apply("TOKEN_REVIEW_AUTHENTICATION"))
```

Comment:
Please look at what Dongbo did in https://github.com/oracle/weblogic-kubernetes-operator/pull/1979. There the pattern is that there are values that can be passed to the Helm chart that preserve the pre-existing behavior. The chart puts the value into the operator's ConfigMap, which can then be accessed through the TuningParameters. Note: the Java code needs to handle the variable's not being present as it won't be present on a Helm upgrade.

Reply:
Just to confirm Ryan, from looking at Dongbo's change, you want the property that controls the new vs old behavior to be defined the values.yaml of the Operator's helm chart and this will then be copied to the Operator's configmap which then can be accessed from TuningParameters? The property won't be defined as an environment variable nor system property.

Reply:
That's correct.

Reply:
Change to use TuningParameters.